server/node/src/main/java/org/kaaproject/kaa/server/admin/client/mvp/view/struct/ConfigFormStructView.java

```diff
@@ -42,7 +42,8 @@ public ConfigFormStructView(HasErrorMessage hasErrorMessage) {
 
     @Override
     protected HasValue<RecordField> createBody(HasErrorMessage hasErrorMessage) {
-        RecordPanel field = new RecordPanel(Utils.constants.configurationBody(), hasErrorMessage, false, false);
+        boolean customLoader = true;
+        RecordPanel field = new RecordPanel(Utils.constants.configurationBody(), hasErrorMessage, false, false, customLoader);
         field.getRecordWidget().setForceNavigation(true);
```

Comment:
Use RecordPanel.setFormDataLoader instead of using addition boolean flag 'customLoader'.